templates/org/menu.tmpl

```diff
@@ -14,6 +14,9 @@
 		{{if and .IsPackageEnabled .CanReadPackages}}
 		<a class="item" href="{{$.Org.HomeLink}}/-/packages">
 			{{svg "octicon-package"}} {{.locale.Tr "packages.title"}}
+			{{if .PackageCount}}
```

Comment:
`NumXXXX` `XXXXCount` which is better?
(not a block, just a question)

Reply:
I think they are both equal